src/main/java/me/xmrvizzy/skyblocker/utils/SlayerUtils.java

```diff
@@ -0,0 +1,79 @@
+package me.xmrvizzy.skyblocker.utils;
+
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.network.ClientPlayerEntity;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.decoration.ArmorStandEntity;
+import net.minecraft.scoreboard.Scoreboard;
+import net.minecraft.scoreboard.ScoreboardObjective;
+import net.minecraft.scoreboard.ScoreboardPlayerScore;
+import net.minecraft.scoreboard.Team;
+
+import java.util.List;
+
+//TODO Slayer Packet system that can provide information about the current slayer boss, abstract so that different bosses can have different info
+public class SlayerUtils {
+    //TODO: Cache this, probably included in Packet system
+    public static List<Entity> GetEntityArmorStands(Entity entity)
+    {
+        return entity.getEntityWorld().getOtherEntities(entity, entity.getBoundingBox().expand(1F, 2.5F, 1F), x-> x instanceof ArmorStandEntity && x.hasCustomName());
+    }
+
+    //Eventually this should be modified so that if you hit a slayer boss all slayer features will work on that boss.
+    public static Entity GetSlayerEntity()
+    {
+        if (MinecraftClient.getInstance().world != null) {
+            var nearestEntities = MinecraftClient.getInstance().world.getEntities();
+            for (var entity : nearestEntities)
+            {
+                if(entity.hasCustomName()) {
+                    //Check if entity is Bloodfiend
+                    if(entity.getCustomName().getString().contains("Bloodfiend"))
+                    {
+                        //Grab the players username
+                        var siblings = MinecraftClient.getInstance().player.getDisplayName().getSiblings();
```

Comment:
If you need the player's username you should instead be doing `MinecraftClient#getSession#getUsername`